src/nvim/msgpack_rpc/remote_ui.c

```diff
@@ -66,8 +66,10 @@ static Object remote_ui_attach(uint64_t channel_id, uint64_t request_id,
       || args.items[2].type != kObjectTypeBoolean
       || args.items[0].data.integer <= 0 || args.items[1].data.integer <= 0) {
     api_set_error(error, Validation,
-                  _("Arguments must be a pair of positive integers "
-                    "representing the remote screen width/height"));
+                  _("Arguments must be:\n"
+                    "- remote screen width (integer > 0)\n"
+                    "- remote screen height (integer > 0)\n"
+                    "- whether to enable RGB capabilities (boolean)"));
     return NIL;
```

Comment:
How about: `"Invalid arguments. Expected: (uint width, uint height, bool enable_rgb)"`.

Also, any change in wording should be reflected in the similar message here: https://github.com/carlosdcastillo/neovim/blob/stringfix/src/nvim/msgpack_rpc/remote_ui.c#L131-L133